wordpress-mobile/WordPress-iOS
# Modularize app tracking

The app delegate was taking care of some app tracking tasks that did not belong to `WPAnalytics`.  Since they do not belong to the app delegate either, I created a new class to take care of it.

**Test 1:**
- Run the unit tests

**Test 2:**
- Set a breakpoint in `WPAppAnalytics - trackApplicationOpened`.
- Set a breakpoint in `WPAppAnalytics - trackApplicationClosed`.
- Run the app, and make sure both are triggered.
- `po analyticsProperties` in both to make sure the data is right.

/cc @jleandroperez 

PS: Travis seems to be still failing so you'll have to check this locally.
